components/developer/rust/Makefile

```diff
@@ -64,7 +64,7 @@ CARGO_HOME=             $(BUILD_DIR)/$(MACH64)
 
 PATH=                   $(CARGO_HOME)/bin:$(GCC_BINDIR):$(PATH.gnu)
 
-#  workaround the symlink name lenght 100 problem, but hope it is not neccessary with
+# workaround the symlink name lenght 100 problem, but hope it is not neccessary with
 # but also with patch src_tools_rust-installer_src_generator.rs.patch 
 COMPONENT_BUILD_ENV+=	CARGO_HOME=$(CARGO_HOME)
```

Comment:
There are some strange things in the `Makefile`.  Something is added to the `COMPONENT_BUILD_ENV` here...